### PR TITLE
Get rid of Travis CI from the docs

### PR DESCRIPTION
**Pull request by potiuk**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by kaxil — COMMENTED** [review]
We still have some tests running in Travis isn't it? Should we wait till we port everything?

**Comment by kaxil** [comment]
Also we have some mentions in the following places too:

- https://github.com/apache/airflow/blob/670118f04a6908c4b088bfca4f99c88bbe696262/CONTRIBUTING.rst 
 - https://github.com/apache/airflow/blob/ac90633fd950c2ea0d45e9f4c4a3d5e024d4b9b6/scripts/ci/ci_build_dockerhub.sh

- https://github.com/apache/airflow/blob/6aa4f534ce6131dd15a8ee2469bf009dec14c87a/STATIC_CODE_CHECKS.rst

- https://github.com/apache/airflow/blob/cf6c254ebdac7d2530d0b9346ca959c47d79a11b/IMAGES.rst

- https://github.com/apache/airflow/blob/cf6c254ebdac7d2530d0b9346ca959c47d79a11b/BREEZE.rst

**Comment by potiuk** [comment]
> Also we have some mentions in the following places too:

Yeah.  Good Catch. I had them all cleaned up in the #8393  but now I split it into smaller pieces. I will see if I can remove also those references without bringing any of the other changes. It was far easier to remove everything and replace it with a new version everywhere when it was one commit/

**Comment by potiuk** [comment]
Fixed @kaxil :). Thanks for the hawk-eye

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -279,8 +277,8 @@ Benefits:
     where all these services are available and can be used by tests
     automatically.
 
--   Breeze environment is almost the same as used in `Travis CI <https://travis-ci.com/>`__ automated builds.
-    So, if the tests run in your Breeze environment, they will work in Travis CI as well.
+-   Breeze environment is almost the same as used in the  CI automated builds.
```
Comment (kaxil) [inline CONTRIBUTING.rst]:
```suggestion
-   Breeze environment is almost the same as used in the CI automated builds.
```

**Review by kaxil — COMMENTED** [review]
1 minor doc change suggested but LGTM otherwise.

**Comment by potiuk** [comment]
Committed :)

**Review by turbaszek — APPROVED** [review]

**Pull request merged**